pkg/webservice/handlers.go

```diff
@@ -918,3 +918,61 @@ func getEvents(w http.ResponseWriter, r *http.Request) {
 		buildJSONErrorResponse(w, err.Error(), http.StatusInternalServerError)
 	}
 }
+
+func getStream(w http.ResponseWriter, r *http.Request) {
```

Comment:
We should limit the number of streams that we allow to be open at the same time here. Even if it is some "high" number but we need to be able to turn down new requests as a safety mechanism.

Reply:
Yes, there is a follow up ticket for that: https://issues.apache.org/jira/browse/YUNIKORN-2147